### DISCUSSION TITLE
How to change truck warp

### DISCUSSION THREAD

**Discussion by Seklose** [haven1433/HexManiacAdvance#201] [Q&A]
Hi, I've been playing around with hexmaniac and changed the map of littleroot so that the houses are in diffrent places. Anyway I can change where I get out of the truck. I checked the truck's warp and found Target(bank, map) is 127,127. Any idea what to change? Thanks!

Here's what I'm looking at:

![image](https://github.com/haven1433/HexManiacAdvance/assets/134235690/e68122ce-7c8b-4324-9d7d-046418029aa4)

**Comment by ShinyTillDawn** [comment]
You have to change the script in the green script tiles. I believe there is a `warpT` command (where `T` is some number) that takes you to map bank 127 map 127 warp 127. You can change those numbers so that the warp tiles take you to a different map.
